content/en/docs/tasks/tools/install-kubeadm.md

```diff
@@ -84,8 +84,9 @@ documentation for the plugins about what port(s) those need.
 ## Installing Docker
 
 On each of your machines, install Docker.
-Version v1.12 is recommended, but v1.11, v1.13 and 17.03 are known to work as well.
+Version v17.03 is recommended, but v1.11, v1.12 and v1.13 are known to work as well.
```

Comment:
Should there actually be a `v` prefix here?

Reply:
i should probably remove the prefix, because the sentence starts with `Version`.